LeaderboardBackend/Models/Entities/Leaderboard.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace LeaderboardBackend.Models.Entities;
+
+public class Leaderboard
+{
+	public long Id { get; set; }
+
+	[Required] 
+	public string Name { get; set; } = null!;
+
+	[Required] 
+	public string Slug { get; set; } = null!;
+
+	public string? Rules { get; set; }
+
+	[JsonIgnore] 
+	public List<Ban>? Bans { get; set; }
+
+	[JsonIgnore] 
+	public List<Category>? Categories { get; set; }
+
+	[JsonIgnore] 
+	public List<Modship>? Modships { get; set; }
```

Comment:
Kinda feel like we should initialise these to be empty Lists instead of nullable ones after all. Would be easier to manage. What do you think?

Reply:
Discussed offline but copying the reasoning for not doing this here:

An empty list would be more confusing because empty list and null would be conflated. They mean different things:
* null = we haven't fetched the relationship
* empty list = there are no relationships
Those two things need to be different, so I think nullable list is right